src/robusta/integrations/kubernetes/process_utils.py

```diff
@@ -25,7 +25,7 @@ class ProcessFinder:
     def __init__(
         self, pod: RobustaPod, filters: ProcessParams, process_type: ProcessType
     ):
-        if process_type != ProcessType.PYTHON:
+        if process_type not in {ProcessType.PYTHON, ProcessType.JAVA}:
```

Comment:
The changes here are good and clean.

Note that for Java processes we're still calling `pod.get_processes()` below which uses the standard `debug-toolkit` and not the Java version. 

That is OK! No reason to change it. But does it mean we can remove duplicate code like 
 `pod_ps` from the `java-toolkit` because we don't need it there?

Reply:
yea added issue to remove duplicate code and combine the toolkits